Sources/StreamVideo/WebRTC/WebRTCClient.swift

```diff
@@ -1024,12 +1024,11 @@ class WebRTCClient: NSObject, @unchecked Sendable {
         for (_, value) in callParticipants {
             if value.id != sessionID {
                 if value.hasVideo {
-                    log.debug("updating video subscription for user \(value.id) with size \(value.trackSize)", subsystems: .webRTC)
+                    log.debug("updating video subscription for user \(value.name) with size \(value.trackSize)", subsystems: .webRTC)
                     var dimension = Stream_Video_Sfu_Models_VideoDimension()
+                    dimension.height = UInt32(value.trackSize.height)
+                    dimension.width = UInt32(value.trackSize.width)
```

Comment:
we remove the scale stuff? Will it have impact on the thermal state?

Reply:
It won't as now we hide stuff when they are not in the viewport (which is the biggest improvement we've made). This is actually something that the SFU should control and will do in the future (once we have a way to pass the device's state so it can make a decision).